iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionContextsProvider.cpp

```diff
@@ -49,11 +49,35 @@ struct CommonRulesEngineSymbolsProvider : IECSymbolProvider
         evalResult.SetValueList(*IValueListResult::Create(values));
         return ExpressionStatus::Success;
         }
+
+    /*---------------------------------------------------------------------------------**//**
+    * @bsimethod
+    +---------------+---------------+---------------+---------------+---------------+------*/
+    static ExpressionStatus GuidToStr(EvaluationResult& evalResult, void*, EvaluationResultVector& args)

```

Comment:
Please add unit tests for this

Reply:
Added unit tests when binary property is a `valid guid`, `invalid guid` and `null`